specification/dimensions/servicecategory.md

```diff
@@ -47,6 +47,7 @@ Allowed values:
 | Security                  | Security monitoring and compliance services.                                                   |
 | Storage                   | Storage services for structured or unstructured data.                                          |
 | Web                       | Services enabling cloud applications to interact via the Internet.                             |
+| Support                   | Platform support related services                                                              |
```

Comment:
We need a full sentence, and it would be best to avoid use of the term "support" when defining this.  Something like this:

```suggestion
| Support                   | Platform assistance related services.                                                             |
```

Reply:
We have similar approach across other services where description uses the cost category term, like Storage or Security etc. I think description here is not necessarily definition of the term, but more like expansion of category. Maybe we could phrase it like 'Platform customer support related services'?

Reply:
That would be fine with me.  But maybe to make it a little more English-readable:

```suggestion
| Support                   | Platform services related to customer support.                                                             |
```